lib-database/src/commonMain/kotlin/com/ustadmobile/lib/db/entities/Site.kt

```diff
@@ -60,10 +65,16 @@ class Site {
 
     var authSalt: String? = null
 
+    @ColumnInfo(defaultValue = "0")
+    var bottomNavVisibilityFlag: Long = 0
```

Comment:
Constants for flags should be on this companion object

Reply:
Should be an int - 32 potential values is more than enough

Reply:
Kept as long because other functions to handle permission constants etc use Long type.